PWGLF/TableProducer/cascqaanalysis.cxx

```diff
@@ -342,7 +343,7 @@ struct cascqaanalysis {
         registry.fill(HIST("hCandidateCounter"), 2.5); // passed topo cuts
         // Fill table
         if (fRand->Rndm() < lEventScale) {
-          mycascades(casc.globalIndex(), collision.posZ(), collision.centFT0M(), collision.centFV0A(), casc.sign(), casc.pt(), casc.yXi(), casc.yOmega(), casc.eta(),
```

Comment:
Hi, indeed, I think this might be wrong. Should be the index of the collision in the collision table.
Are you sure that you don't need indices?

Reply:
Hi, Nicolò! We got errors during the merging of AO2Ds that we had Index on O2collision and O2mccollision tables but no trees were found. Anton said that this error means that "the merger cannot find the target tree of the index and thus cannot properly offset the index based on that tree size. Standalone derived data either should not have indices pointing to original AODs or should not be merged", so we decided to remove them at all, although I think in future it might be usefull to know at least whether the candidates are from the same collision or not.

Reply:
I see, indeed if you don't produce the collision it should not matter, for future `mycascades(casc.globalIndex()` still should be `mycascades(tableColl.lastIndex()` or so